### PR TITLE
feat(withdrawal): print txhash of withdrawal

### PR DESCRIPTION
**Pull request by golangisfun123**
**Description**
A clear and concise description of the features you're adding in this pull request.

**Additional context**
Add any other context about the problem you're solving.

**Metadata**
- Fixes #[Link to Issue]

## Summary by CodeRabbit

- **New Features**
  - Introduced a new method to retrieve transaction hashes by nonce.
  - Added spinner for transaction status in the RFQ relayer.

- **Bug Fixes**
  - Updated dependencies and imports across multiple modules to improve stability.

- **Tests**
  - Added test functions for Ethereum and ERC20 token withdrawals.
  - Updated test imports for assertion libraries.

- **Chores**
  - Standardized and reorganized imports for better code maintainability.
  - Commented out the `editor.formatOnSave` setting in the VSCode configuration.

### CODE REVIEW

File: core/metrics/metrics_test.go [hunk 1]
```diff
@@ -2,12 +2,13 @@ package metrics_test
 
 import (
 	"context"
-	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/alecthomas/assert"
```
Comment (trajan0x) [inline core/metrics/metrics_test.go:7]: what's the point of this?

**Review by trajan0x — CHANGES_REQUESTED** [review]

File: core/metrics/metrics_test.go (hunk 1 above)
Comment (golangisfun123) [reply core/metrics/metrics_test.go]: if i edit the file, undo the changes, then save, gopls still reorders the imports

Comment (trajan0x) [reply core/metrics/metrics_test.go]: btw I meant using 	"github.com/alecthomas/assert" instead of 	"github.com/stretchr/testify/assert"

Comment (golangisfun123) [reply core/metrics/metrics_test.go]: Wrong import inference I think, wasn't intentional. Removed now

**Pull request merged**